templates/admin/dashboard.tmpl

```diff
@@ -76,7 +76,7 @@
 			{{ctx.Locale.Tr "admin.dashboard.system_status"}}
 		</h4>
 		{{/* TODO: make these stats work in multi-server deployments, likely needs per-server stats in DB */}}
-		<div hx-get="{{$.Link}}/system_status" hx-swap="morph:innerHTML" hx-trigger="every 5s" hx-indicator=".divider" class="ui attached table segment">
+		<div hx-get="{{$.Link}}/system_status" hx-swap="morph:innerHTML" hx-trigger="every 5s" hx-indicator=".tw-hidden" class="ui attached table segment">
```

Comment:
```suggestion
		<div hx-get="{{$.Link}}/system_status" hx-swap="morph:innerHTML" hx-trigger="every 5s" class="ui attached table segment">
```

Will this not work?

Reply:
This will place the indicator on this div

Reply:
Is the indicator placement default behavior of htmx or do we configure that somewhere?

Reply:
Or maybe a `data-indicator="initial-only"`.

Reply:
Here https://github.com/go-gitea/gitea/blob/993736d838c36e26951b6cfea9c6a549958addd1/web_src/js/htmx.js#L8

But we shouldn't delete it, because we need it for subscribe and watch

Reply:
Ok I see so normally htmx would add a different class, but we replace that. It seems htmx offer no API to disable this behaviour.

So I would say remove the `hx-indicator` attribute and add a CSS rule to suppress showing the spinner via CSS:

```css
.unique-div-class.is-loading::after {
  content: none;
}
```

Reply:
BTW, what's wrong exactly with rendering a indicator in place where content will appear, e.g. on the parent div?

Reply:
> add a CSS rule

I prefer the current way

> what's wrong exactly with rendering the spinner in place where content will appear, e.g. on the parent div?

It makes the entire div a bit darker

Reply:
IIRC, it only set's `opacity` of foreground content to 0.3 currently, I would personally prefer it to be 0 even.